src/LeanCode.ContractsGenerator/Analyzers/InvalidTypeCheck.cs

```diff
@@ -1,25 +1,38 @@
 namespace LeanCode.ContractsGenerator.Analyzers;
 
-public class InvalidTypeCheck : BaseAnalyzer
+public abstract class InvalidInternalTypeCheck(string name, string message) : BaseAnalyzer
```

Comment:
Fwiw this type is now unused, so if anyone feels that We Aren't Going to Need It, feel free to tell me to remove it.